### PR TITLE
[NIC-27] Add get logic document-service

### PR DESCRIPTION
**Pull request by AksyonovaKaterina**

### CODE REVIEW

File: backend/document-service/src/main/kotlin/com/nosql/document/controller/DocumentController.kt [hunk 1]
```diff
@@ -21,6 +25,30 @@ class DocumentController(
     private val documentService: DocumentService,
 ) {
 
+    @GetMapping(
+        produces = [APPLICATION_JSON_VALUE],
+    )
+    suspend fun getAllByUserId(
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/controller/DocumentController.kt]: maybe use `/all` path?

File: backend/document-service/src/main/resources/http/documentControllerRequests.http [hunk 2]
```diff
@@ -7,4 +7,22 @@ Content-Type: application/json
...
+Content-Type: application/json
+
+### Get all watcher
+GET http://localhost:8083/public/api/v1/document/all?page=0
+#Authorization: Bearer <access_token>
+Authorization: Bearer eyJraWQiOiIxNDQwNzcyMTY5NDQyNDUwMjM2NCIsImFsZyI6IlJTMjU2In0.eyJpc3MiOiJub3NxbC1pdC1jb21wYW55Iiwic3ViIjoiNjM4NTFmYTViN2YzZGEwOTExYjgxZWQ1IiwiaWF0IjoxNjY5NjczNzExLCJleHAiOjE2ODUyMjU3MTEsInJvbGUiOiJIUiJ9.JdDF9BfqZoEFnplSEXKlgJRjYxGmW5EqLjAAay7Z3N1hE0eJ1nfeAs3n9ERcjRX2ScbUkY4bqpfawP8AZbg6lmZSn1_WYtwmLesVhPlesJdo7U9plBwWIJth1GLMa4qJpOzNHaVIST7un7sT_5q2shnb9K1BQacwv4p0SrE2mpM_PDiePN-4ZYYG7RXNC2lbAJsTyY7lMTH4nl620ve8qComDL4bbaSsr7lnUPz-4SmSYKHSqEP952WKlbxrBA2WlnJE3lSgXYIxu5NuLmu7TSexOjD4Y-ykP0d3Y8r1xBZk_2BZijpR2I1_Mjfg7fTdtxVxe0eZlexebZXdZ7J5Mw
```
Comment (V7074) [inline backend/document-service/src/main/resources/http/documentControllerRequests.http]: token?

**Review by V7074 — CHANGES_REQUESTED** [review]

File: backend/document-service/src/main/resources/http/documentControllerRequests.http (hunk 2 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/resources/http/documentControllerRequests.http]: fixed

File: backend/document-service/src/main/kotlin/com/nosql/document/controller/DocumentController.kt (hunk 1 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/controller/DocumentController.kt]: fixed

**Review by V7074 — APPROVED** [review]

**Pull request merged**